conf/playReleases.json

```diff
@@ -1,14 +1,10 @@
 {
   "latest": {
-    "version": "2.8.10",
-    "date": "Nov 30 2021"
+    "version": "2.8.11",
+    "date": "Dec 1 2021"
   },
   "development": [],
   "previous": [
-    {
-    "version": "2.8.9",
```

Comment:
Was this meant to be removed?  Or should `2.8.10` have been added, instead?

Reply:
Removed from the website and from the release page in GitHub. They are still on maven central. We can't remove from there.

Reply:
For more context...

2.8.9 was released last week, but we had an issue with play-ws and scala-java8-compat

Yesterday, I released 2.8.10 and updated the website. Just to discover that the build was done with jdk11. My bad. 

The three version are the same in functionality, but only 2.8.11 can be used with jdk8 and use play-ws with sbt 1.5. 

So, advertising 2.8.9 and 2.8.10 isn't very helpful for the users. People should just move to 2.8.11.